### ISSUE TITLE
This MOD on Steam Deck

### ISSUE THREAD

**Issue by rdlf4** [Hary309/hry-sic#14]
Hello there.

I've followed the instructions provided in this GitHub, downloaded the zip file **hry-sic-2.0.6-manual-installation** and extracted the 'plugins' folder from the package to _steamapps/common/Euro Truck Simulator 2/bin/win_x64_ directory on my Steam Deck.
Upon entering the game, I get the "CONFIRMATION" pop up window, letting me know a request to use advanced SDK features was detected. I get to drive my truck but the plugin doesn't work. I'm currently delivering a package in World of Trucks, so I'm not sure if that's the reason, will come back to report if the same occurs when playing offline. 

I remember this Github mentioned Delete summons the plugin window, which it doesn't. I used a Steam overlay to get to the controller settings of this game. I edited the controller layout and assigned R4 to Delete. When pressing that while I'm in the cabin and the truck is off, nothing happens. With the truck operating, Delete only changes some information related to gas consumption on the panel. I went to controller settings but there's no option there that's assigned Delete to.
Also, I've checked the MODs for this game from the game's UI. This camera mod isn't listed there, but I guess it's because it only displays mods from Steam Workshop.

So right now, I have no idea how to get this mod to work, nor how do display this mod's settings, all I know is it doesn't work out of the box, after extracting it to /bin/win_x64 directory.

Additional info:
*SteamOS ver 3.4.11
*I'm launching this game with DirectX11(win64)
*I'm also trying launching this game with OpenGL, just in case. No dice.
*No error messages have popped up
*I STILL haven't found where to change the button to make this plugin's menu to show.
As stated by this Github:
```To open Main Window press Delete (you can change this bind in Key binds section)```

Any help is appreciated. Thank you.

**Comment by rdlf4** [comment]
Yeah I can confirm this mod has no effect on v1.48. Tested it online & offline. Customized controller layout (driving section) to assign DELETE to one of Deck's back buttons, and then pressed it to no effect. The only thing that happens is the game recognizes something is requesting advanced SDK features at start of the game, but that's about it, sadly. I found no other way to edit a file so I could use another button other than Delete to have a look at the plugin's settings UI, but I ended up getting frustrated because I believe I was looking for something that's not there. Not really sure what the developer meant with ```To open Main Window press Delete (you can change this bind in Key binds section)```

Anyway, that's it for this plugin if you own a Steam Deck.

**Comment by Hary309** [comment]
Thanks for checking. I don't have a Steam Deck and couldn't verify it. I am sorry to hear that but currently, the project is in limbo - I had a tough time for the last two years (studies, work, problems with mental health) but I am getting better and I have some big ideas to return with.
